### ISSUE TITLE
Batch Download

### ISSUE THREAD

**Issue by Harjuuso** [subhra74/xdm#1238]
## Checklist

- [x ] I'm asking a question
- [x ] I've looked through the README and FAQ for similar questions
- [x ] I've searched the bugtracker for similar questions including closed ones

## Question

Batch download not working, I assume. What format should I put the links in? Like; link "enter" link "enter"? Doesn't seem to work that way: 
![image_2024-06-28_234750015](https://github.com/subhra74/xdm/assets/69907019/62e8ec46-01b0-4db7-aba7-dd88ad582a9e)

**Comment by inuonly** [comment]
yes batch download is not working i can confirm too and please don't label as question. This is a bug/issue . Multi link downloads are not working in batch download menu of the program. The normal jpg links are also not getting downloaded. I tried this on bing wallpapers github repo links and when you press download it does nothing the window reamins as is.

**Comment by Harjuuso** [comment]
Alright! It was question because I was unsure of how to use it, but turns out its actually broken.. thank you for your response!

seems like i can’t change the label..?

**Comment by inuonly** [comment]
> Alright! It was question because I was unsure of how to use it, but turns out its actually broken.. thank you for your response!
> 
> seems like i can’t change the label..?

I think that is moderator/admin's job. Also just to be clear, I was not calling you out. Instead of creating new issue I posted here so that this is setup as new issue for the repo. 
Have a good one ahead!

**Comment by JTCHE** [comment]
bump, seems like this feature still isn't properly working as of today :/ does it expect a specific delimiter other than line breaks ?
